components/ai_chat/core/browser/model_service.cc

```diff
@@ -253,6 +253,232 @@ const std::vector<mojom::ModelPtr>& GetLeoModels() {
       models.push_back(std::move(model));
     }
 
+    // Llama 4 Scout
+    {
```

Comment:
This is very verbose, you could either pass the properties to the `New` call, create a method to encapsulate creating them, etc...

Reply:
I'm not much of a c++ guy so I was following the existing format, didn't wanna rock the boat. @Douglashdaniel wdyt?

Reply:
could be a follow-up as well